changelog.md

```diff
@@ -5,6 +5,14 @@
 ### Changes
 
 - [#4149](https://github.com/ignite/cli/pull/4149) Bump cometbft to `v0.38.7`
+- [#4168](https://github.com/ignite/cli/pull/4168) Bump IBC to `v8.3.1`
+  If you are upgrading manually from `v8.2.0` to `v8.3.1`, add the following to your `ibc.go` file:
```

Comment:
Should I instead list the command to use with `gen-mig-diffs`?
Supposedly: `gen-mig-diffs --output changes.md --from v28.4.0 --to v28.5.0`

Reply:
cc @toschdev. do we have the docs / tutorials published for this? so we could link this insteand